### PR TITLE
add name

### PR DESCRIPTION
**Pull request by SantaSati**
try

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,9 +125,14 @@ Please do NOT edit or remove others' link.
 - Wu Xinran
 - Yuan Wenxuan
 - Mayi
+- GuoYao
+
+ qwe
+=======
```
Comment (wanghaiwei) [inline README.md]: please make sure you slove conflict right

**Comment by wanghaiwei** [comment]
please make sure you slove conflict right

**Comment by Swinggcat** [comment]
> please make sure you slove conflict right

Please make sure you have resolved those conflicts correctly

**Comment by Swinggcat** [comment]
Done. I've done that.

**Pull request merged**